docs/docs/04-quickstartlocal.md

```diff
@@ -4,7 +4,52 @@ title: Quick Start Local
 ---
 
 # QuickStart with Local Deployment
```

Comment:
I think we should separate out the stuff about setting up your dev env from the stuff that's required just to run the app locally. Keep the basic stuff here in QuickStart Local but move all the dev env stuff to another file (or back to the readme).